cpp/ql/src/change-notes/2023-07-03-improve-uninitialized-local.md

```diff
@@ -0,0 +1,4 @@
+---
+category: majorAnalysis
```

Comment:
```suggestion
category: minorAnalysis
```
I don't think this warrants a `majorAnalysis` tag. A `majorAnalysis` change bumps the minor version-number, which I think is a bit too much for this change.